### DISCUSSION TITLE
No Longer Usable - Mac 4.13.7 (21182)

### DISCUSSION THREAD

**Discussion by DeanMultimedia** [iterate-ch/cyberduck#14496] [Q&A] [answered]
On a Mac. Server is DreamHost.

The latest update is failing to save files properly direct from my editing software (CotEditor).

EVERY time I save, I'm notified by CotEditor that the file's been modified by another app, which simply means CotEditor is still trying to save the file from a previous attempt.

CotEditor is not the problem. It's informing me that Mountain Duck is.

So here I am.

**Comment by dkocher** [comment]
I cannot reproduce this using _CotEditor Version 4.5.2 (562)_ when saving a plain text file to a volume connected to WebDAV or SFTP.

**Comment by DeanMultimedia** [comment]
Mac OS 13.3 (22E252), CotEditor 4.5.2 (562), Mountain Duck 4.13.7 (21182)

SERVER: Dreamhost

**Well, current status goes something like this.**

1) Prereq: computer is rebooted, network is reset, outbound firewall disabled (though MD was allowed)
2) Open Mountain Duck
3) Connect to bookmark that was working normally

- It took longer than usual, for sure (7 ~ 10 seconds)
- Traverse folders to PHP file, open
- make minor edit (add space character, save)

**File does NOT save. No "Date Modified" change.**

... but I bet it's "trying" to.

So I hit "command S" again. SURE ENOUGH:

"This document’s file has been changed by another application since you opened or saved it. The changes made by the other application will be lost if you save. Save anyway?"

**So then I WAR SAVE it.** Hold down command "S" while repeatedly saying "save" to the above constantly re-appearing prompt.

It saves. Eventually.

Then, my log blows up.

[log.txt](https://github.com/iterate-ch/cyberduck/files/11180059/log.txt)

Meanwhile, Cyber DUCK is working. I think. But I didn't pay for that. I paid for Mountain.

**Comment by dkocher** [comment, marked as answer]
We have identified the issue and are tracking this in #14412.

**Comment by DeanMultimedia** [comment]
Maybe it's conjoined to this bug and maybe it's not but ...

One new thing that's happened since the (unusable) update, my Mac now shows TWO mounted volumes for any given bookmark.

The first "disk" is as expected, namely the hd icon that you select and peruse.

The SECOND, however ... is a network drive, which begs the question "why two connections?"

See attachment for dual connectivity. Note the three "TRO" directories. One is a local folder. Ignore that. But as for the "drives" (far top and far bottom) ...

It makes no difference which I select/traverse. Both load but neither are capable of saving any of my work.

And it is work, so if we could get a little more interaction on what actions are being taken to resolve this, my workflow will once again be efficient.

<img width="85" alt="screenshot-90" src="https://user-images.githubusercontent.com/46755854/232098515-f30a77c6-6793-4525-b2a0-2f004469d75a.png">

**Reply by dkocher** [reply]
> Maybe it's conjoined to this bug and maybe it's not but ...

This is unrelated to the original discussion. macOS Ventura 13.3 or later will always display an item in _Locations_ for the mounted volume in addition to the item in _Favourites_ added by Mountain Duck. You can configure to hide _Locations_ in _Finder → Settings… → Sidebar_.
